deploy/runtime/help/html/battery_dispatch_fom.htm

```diff
@@ -47,7 +47,7 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Initial state of charge, %</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">The state of charge of the battery at the beginning of the simulation. A value of 100% would be for a fully charged battery, 0% would be for a fully discharged battery.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Minimum time at charge state, min</span></p>
```

Comment:
In this file, not this line: PV smoothing _is_ available for generic system-battery (which is appropriate), but the help note says it's not.